next-cloudinary/package.json

```diff
@@ -2,13 +2,30 @@
   "name": "next-cloudinary",
   "version": "6.13.0",
   "license": "MIT",
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
-  "source": "src/index.ts",
+  "main": "dist/next-cloudinary.cjs.js",
+  "module": "dist/next-cloudinary.esm.js",
+  "exports": {
+    ".": {
+      "types": {
+        "import": "./dist/next-cloudinary.cjs.mjs",
```

Comment:
curious why these aren't referring to a .d.ts file?

Reply:
I don't recall exactly but it might have its origins in https://github.com/microsoft/TypeScript/issues/5438

This is fine though as TS always tries to load the respective declaration file for any JS source file through, what I I'm calling, "a sibling lookup". Essentially it just replaces the extension and tries to load that instead with a higher priority.